projects/ccf-rui/src/app/shared/components/labeled-side-toggle/labeled-side-toggle.component.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { Shallow } from 'shallow-render';
+
+import { LabeledSideToggleComponent } from './labeled-side-toggle.component';
+import { LabeledSideToggleModule } from './labeled-side-toggle.module';
+
+describe('LabeledSideToggleComponent', () => {
+  let shallow: Shallow<LabeledSideToggleComponent>;
+
+  beforeEach(() => {
+    shallow = new Shallow(LabeledSideToggleComponent, LabeledSideToggleModule);
+  });
+
+  it('should emit the correct gender whenever updateGender is called.', async () => {
+    const { instance, outputs } = await shallow.render({ bind: { isMale: true } });
+    instance.updateGender(true);
+    expect(outputs.genderChanged.emit).toHaveBeenCalledWith('female');
```

Comment:
Both tests should be split into two separate ones. One testing the positive case and one testing the negative case. Having more than one `expect` statement is usually an indicator that it needs to be split.